server/src/test/java/org/elasticsearch/index/shard/IndexShardTests.java

```diff
@@ -1319,7 +1319,7 @@ public void testRefreshMetric() throws IOException {
         }
         indexDoc(shard, "_doc", "test");
         shard.writeIndexingBuffer();
-        assertThat(shard.refreshStats().getTotal(), equalTo(refreshCount+2));
+        assertThat(shard.refreshStats().getTotal(), greaterThan(refreshCount));
```

Comment:
Not sure whether we can assert that this will be 1 greater rather than 2, or if the change in writeIndexingBuffer adds an element of randomness here that means we can only assert that it will be larger.